mediapipe/tasks/ios/vision/core/sources/MPPMask.h

```diff
@@ -0,0 +1,118 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** The underlying type of the segmentation mask. */
+typedef NS_ENUM(NSUInteger, MPPMaskDataType) {
+
+  /** Represents the native `UInt8 *` type. */
+  MPPMaskDataTypeUInt8,
+
+  /** Represents the native `float *` type. */
+  MPPMaskDataTypeFloat32,
+
+} NS_SWIFT_NAME(MaskDataType);
+
+/**
+ * The wrapper class for MediaPipe segmentation masks.
+ *
+ * Masks are stored as `UInt8 *` or `float *` objects.
+ * Every mask is has an underlying type which can be accessed using `dataType`. You can access the
+ * mask as any other type using the appropriate properties. For eg:, if the underlying type is
+ * `MPPMaskDataTypeUInt8`, in addition to accessing the mask using `uint8Array`, you can access
```

Comment:
Missing "it" or "the data".

We should mention the different value ranges as well.